apps/auth_zero/tests.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from common.config import ConfigFileLoader, CredentialsLoader
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        self.user_model = get_user_model()
+        self.creds_obj = CredentialsLoader(settings.CONFIG_FILE, ConfigFileLoader)
```

Comment:
test_config config path can be moved and hard coded here. we will look at this later; but remove from dev.py and move it right here

Reply:
Done